packages/app-desktop/commands/exportDeletionLogs.ts

```diff
@@ -0,0 +1,42 @@
+
+import { CommandRuntime, CommandDeclaration } from '@joplin/lib/services/CommandService';
+import { _ } from '@joplin/lib/locale';
+import Logger from '@joplin/utils/Logger';
+import { TargetType } from '@joplin/utils/Logger';
+import shim from '@joplin/lib/shim';
+import Setting from '@joplin/lib/models/Setting';
+import bridge from '../services/bridge';
+
+export const declaration: CommandDeclaration = {
+	name: 'exportDeletionLogs',
+	label: () => _('Export deletion logs'),
+};
+
+export const runtime = (): CommandRuntime => {
+	return {
+		execute: async () => {
+			if (!Logger.globalLogger) {
+				throw new Error('Could not find global logger');
+			}
+
+			const fileTarget = Logger.globalLogger.targets().find(t => t.type === TargetType.File);
+
+			if (!fileTarget || !fileTarget.path) {
+				throw new Error('Log file target not found');
+			}
```

Comment:
If the log file was recently rotated, you won't get the information. Maybe instead just directly get log.txt and the latest log-*.txt from the profile directory?